### DISCUSSION TITLE
connect with mentor

### DISCUSSION THREAD

**Discussion by pyther-hub** [weecology/retriever#1662] [Q&A] [answered]
sorry I know what I am doing here is wrong, but I am new to open source and wanted to contribute to the code 
but really don't have any clue on what to do ( have good experience in python and deep learning), I did my research on how to work around in these things (how to contact a mentor) but  it was not helpful, so it would be very kind of you if you can guide me on what to do or just provide me if there is any slack  or discord channel for this

**Comment by Nageshbansal** [comment]
Hey @pyther-hub , you can ask questions on the gitter channel of Weecology. Here I'm dropping the link for the same. 
https://gitter.im/weecology/retriever

**Comment by henrykironde** [comment, marked as answer]
Thanks @Nageshbansal. Hi @pyther-hub Thank you for your interest in open source. I recommend a generic approach to getting started. Select a project that you feel could be interesting, and the first thing would be to go through the README file and then the documentation. Usually, the documentation contains everything you need to get started. As you read through the documents, create issues in case something comes up that is not clear or is out of date.

**Comment by RishithaR-388** [comment]
Hello @henrykironde   , I am Rishitha Reddy from India.I am a 3rd year UG student at IIT Bhilai in DSAI discipline, India. I am proficient at python,machine learning,tensorflow,numpy, pandas and other deep learning frameworks.I have done many projects using deep learning neural networks, transfer learning .One of my projects is facial expression recognition using deep learning and  opencv.I came across the project Multi Class training and prediction in Deep Forest and would like to contribute for this project in the GSoc 2023. I am new to open source and wanted to contribute to the code but really don't have any clue on what to do, so it would be very helpful of you if you can guide me on what issues to be started.

**Reply by henrykironde** [reply]
Please first read the documents for the project you want to work on. Then move on to the developer documentation. In case you have an issue, let us know.https://deepforest.readthedocs.io/en/latest/developer.html.
